### ISSUE TITLE
URL Handler: support untitled files

### ISSUE THREAD

**Issue by MagTun** [microsoft/vscode#74616]
I am able to open current file (at a specified line and column) with :
`vscode://file/C:/Users/Me/Desktop/file.py:11:1`
But I would like to be able to open a new file with something like this:
`vscode://new?text=text_that_will_be_in_file&type=js
`
This was asked [here ](https://github.com/Microsoft/vscode/issues/4883)but apparently not implemented and the issue is closed so I can't comment on it.

**Comment by ozyx** [comment]
What's the use case for this? I'd like to take a stab at implementing this.

**Comment by ozyx** [comment]
@joaomoreno thoughts?

**Comment by MagTun** [comment]
Sorry for not answering before, I totally forgot it.

 I needed this when doing a local html app. The main purpose of that app is to retrieve local scripts and display them inside hierarchical `div` that can be easily hidden/displayed. I would have like to add several texteareas where I could pre-write a missing code and then click on a button that will get the content of the textearea an open it up in VsCode in an Untitled file where I could finish writing it/saving it. 

It didn't get to the step of a very precise feature but as I was searching how I  could implement this I found the thread mentioned in my question and thought it was forgotten in the process, so I reopened a new issue for it.
